src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/EETypeNode.cs

```diff
@@ -76,6 +76,12 @@ public EETypeNode(NodeFactory factory, TypeDesc type)
             _hasConditionalDependenciesFromMetadataManager = factory.MetadataManager.HasConditionalDependenciesDueToEETypePresence(type);
 
             factory.TypeSystemContext.EnsureLoadableType(type);
+
+            // We don't have a representation for function pointers right now
```

Comment:
Is this going to print System.IntPtr like in CoreCLR?
```
unsafe
{
    Console.WriteLine(typeof(delegate* <void>));
}
```

Reply:
No, it didn't do it before this change and it won't do it after. The expected behavior is that we rewrite this method body to a throw...